src/app/src/app.reducers.js

```diff
@@ -49,12 +51,20 @@ const appReducer = createReducer(
                     $set: true,
                 },
             }),
-        [setSensorData]: (state, payload) =>
+        [completePollingSensor]: (state, payload) =>
             update(state, {
                 sensorData: {
                     [payload.id]: {
                         $set: payload,
-                    }
+                    },
+                },
+            }),
+        [updateSensorRatings]: (state, payload) =>
+            update(state, {
+                sensorRatings: {
+                    [payload.id]: {
```

Comment:
Oops! The payload is missing an `id` field, otherwise we get:
<img width="483" alt="screen shot 2019-03-01 at 11 29 46 am" src="https://user-images.githubusercontent.com/10568752/53651854-8d0e1180-3c15-11e9-8564-7dc97d80714d.png">

This may change with #64

Reply:
Oh good catch. Fixed.